tests/csv/adword_keywords.csv

```diff
@@ -0,0 +1 @@
+AWS
```

Comment:
This is a duplicate of the file `fixture/adword_keywords.csv` so it can be removed

Reply:
😮 My bad. Fixed in 4cc2fbcd1abd04500ccc952eec2ae4864c9c9bef